content/id/docs/tasks/administer-cluster/manage-resources/memory-constraint-namespace.md

```diff
@@ -0,0 +1,272 @@
+
+Berikut berkas konfigurasi Pod yang memiliki satu Kontainer. Manifes Kontainer
+menentukan permintaan memori 600 MiB dan limit memori 800 MiB. Nilai tersebut memenuhi
+batasan minimum dan maksimum memori yang ditentukan oleh LimitRange.
+
+{{< codenew file="admin/resource/memory-constraints-pod.yaml" >}}
+
+Membuat Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/admin/resource/memory-constraints-pod.yaml --namespace=constraints-mem-example
+```
+
+Memastikan Kontainer pada Pod sudah berjalan:
+
+```shell
+kubectl get pod constraints-mem-demo --namespace=constraints-mem-example
+```
+
+Melihat informasi mendetail tentang Pod:
+
+```shell
+kubectl get pod constraints-mem-demo --output=yaml --namespace=constraints-mem-example
+```
+
+Keluaran yang dihasilkan menunjukkan Kontainer memiliki permintaan memori 600 MiB dan limit memori
+800 MiB. Nilai tersebut memenuhi batasan yang ditentukan oleh LimitRange.
+
+```yaml
+resources:
+  limits:
+     memory: 800Mi
+  requests:
+    memory: 600Mi
+```
+
+Menghapus Podmu:
+
+```shell
+kubectl delete pod constraints-mem-demo --namespace=constraints-mem-example
+```
+
+## Mencoba membuat Pod yang melebihi batasan maksimum memori
+
+Berikut berkas konfigurasi untuk sebuah Pod yang memiliki satu Kontainer. Kontainer tersebut menentukan
+permintaan memori 800 MiB dan batas memori 1.5 GiB.
+
+{{< codenew file="admin/resource/memory-constraints-pod-2.yaml" >}}
+
+Mencoba membuat Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/admin/resource/memory-constraints-pod-2.yaml --namespace=constraints-mem-example
+```
+
+Keluaran yang dihasilkan menunjukkan Pod tidak dibuat, karena Kontainer menentukan limit memori yang
+terlalu besar:
+
+```
+Error from server (Forbidden): error when creating "examples/admin/resource/memory-constraints-pod-2.yaml":
+pods "constraints-mem-demo-2" is forbidden: maximum memory usage per Container is 1Gi, but limit is 1536Mi.
+```
+
+## Mencoba membuat Pod yang tidak memenuhi permintaan memori
+
+Berikut berkas konfigurasi untuk sebuah Pod yang memiliki satu Kontainer. Kontainer tersebut menentukan
+permintaan memori 100 MiB dan limit memori 800 MiB.
+
+{{< codenew file="admin/resource/memory-constraints-pod-3.yaml" >}}
+
+Mencoba membuat Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/admin/resource/memory-constraints-pod-3.yaml --namespace=constraints-mem-example
+```
+
+Keluaran yang dihasilkan menunjukkan Pod tidak dibuat, karena Kontainer menentukan permintaan memori yang
+terlalu kecil:
+
+```
+Error from server (Forbidden): error when creating "examples/admin/resource/memory-constraints-pod-3.yaml":
+pods "constraints-mem-demo-3" is forbidden: minimum memory usage per Container is 500Mi, but request is 100Mi.
+```
+
+## Membuat Pod yang tidak menentukan permintaan ataupun limit memori
+
+
+
+Berikut berkas konfigurasi untuk sebuah Pod yang memiliki satu Kontainer. Kontainer tersebut tidak menentukan
```

Comment:
Keep the API name "Container"
```suggestion
Berikut berkas konfigurasi untuk sebuah Pod yang memiliki satu Container. Container tersebut tidak menentukan
```

Reply:
All resolved in e323b8c5920d6ae02c050306495e784b8e19ad9f